cmd/ndm_daemonset/controller/controller.go

```diff
@@ -275,10 +275,13 @@ func (c *Controller) Broadcast() {
 }
 
 // run waits until it gets any interrupt signals
-func (c *Controller) run(threadiness int, stopCh <-chan struct{}) error {
+func (c *Controller) run(threadiness int, ctx context.Context) error {
```

Comment:
We are passing channel from [here](https://github.com/openebs/node-disk-manager/pull/561/files#diff-a20906eb0434b5deaf080095f98faa6ab0cd1420cece8129fc0ec7102c14d9f6R259), how will the context work in this case?